extensions/lifecycle/util/rules.js

```diff
@@ -193,6 +194,25 @@ function rulesToParams(versioningStatus, currentDate, bucketLCRules, bucketData,
     return _getParamsFromListings(bucketName, currentDate, listings, options);
 }
 
+/**
+ * Formats the array of supported lifecycle rules in Arsenal so that
+ * they correspond to how they are written in the lifecycle configuration.
+ * @param {string[]} supportedLifecycleRules list of supported lifecycle rules
+ * @returns {string[]} formatted supported lifecycle rules
+ */
+function getFormattedSupportedLifecycleRules() {
+    if (!Array.isArray(supportedLifecycleRules)) {
+        return [];
```

Comment:
can this case actually happen?
we should always have a (non-empty) array I guess....